tests/integration/test_nitrate.py

```diff
@@ -0,0 +1,85 @@
+import yaml
+import shutil
+import tempfile
+from fmf import Tree
+from pathlib import Path
+from click.testing import CliRunner
+from requre import RequreTestCase
+import tmt.base
+import tmt.cli
+
+# Prepare path to examples
+TEST_DIR = Path(__file__).parent
+NITRATE_EXAMPLE = TEST_DIR.parent.parent / "examples" / "nitrate_cases"
+
+
+# general test plan for this component is: TP#29309
+class NitrateExport(RequreTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tmpdir = Path(tempfile.mktemp())
+        shutil.copytree(NITRATE_EXAMPLE, self.tmpdir)
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+        super().tearDown()
+
+    def test_create(self):
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertNotIn("extra-nitrate", fmf_node.data)
+        test = tmt.base.Test(fmf_node)
+        test.export(format_='nitrate', create=True, general=True)
+        # reload the node data to see if it appear there
+        fmf_node = Tree(self.tmpdir).find("/new_testcase")
+        self.assertIn("extra-nitrate", fmf_node.data)
+
+    def test_existing(self):
+        fmf_node = Tree(self.tmpdir).find("/existing_testcase")
+        self.assertEqual(fmf_node.data["extra-nitrate"], "TC#0609686")
+        test = tmt.base.Test(fmf_node)
+        print(test.fmf_id)
+        test.export(format_='nitrate', create=True, general=True)
+
+
+class NitrateImport(RequreTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.tmpdir = Path(tempfile.mktemp())
+        shutil.copytree(NITRATE_EXAMPLE, self.tmpdir)
+
+    def tearDown(self):
+        shutil.rmtree(self.tmpdir)
+        super().tearDown()
+
+    def test_import_manual_confirmed(self):
+        runner = CliRunner()
+        # TODO: import does not respect --root param anyhow (could)
+        result = runner.invoke(
+            tmt.cli.main, ['--root', self.tmpdir / "import_case", "test", "import", "--nitrate", "--manual", "--case=609704"])
+        self.assertEqual(result.exit_code, 0)
+        self.assertIn("Importing the 'Imported_Test_Case'", result.output)
+        self.assertIn("test case found 'TC#0609704'", result.output)
+        self.assertIn("Metadata successfully stored into", result.output)
+        filename = next(filter(lambda x: "Metadata successfully stored into" in x and "main.fmf" in x,
```

Comment:
Could you please break long lines and keep 79 chars max as we do for all python scripts? Thanks.